R/methods.R

```diff
@@ -255,10 +255,45 @@ setMethod("aggregate_cells", "SingleCellExperiment",  function(.data,
   se <- lapply(selected_experiments_list, aggregate_assays_fun) |> 
     purrr::reduce(full_join) |> 
     suppressMessages()
-  if(length(selected_experiments_list) >1 ) warning("tidySingleCellExperiment says: Features from all selected assays have been combined!")
-  se |> 
+  
+  if(se |> 
+      distinct(assay_type, .feature) |> 
+      pull(.feature) |> 
+      duplicated() |> 
+      any()) {
```

Comment:
Since we are using this condition twice, to make the code less bug prone for future developers.

1) let's make it a variable 

```r
are_feature_duplicated_across_assays = 
se |> 
      distinct(assay_type, .feature) |> 
      pull(.feature) |> 
      duplicated() |> 
      any()
```

2) use this variable at line 259, and in line 279 (if I understand, well, they are checking the same condition)